src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/Editor/DefaultTextBufferProjectService.cs

```diff
@@ -24,13 +24,16 @@ internal class DefaultTextBufferProjectService : TextBufferProjectService
 
```

Comment:
Technically didn't need to touch this file, it's just cleanup to re-use our capability resolver infrastructure.